script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1673,7 +1673,7 @@ Thank goodness... She's back to being her usual self.
 -- 「そ、そうですか。奇遇ですが、わ、わたしも同じですので、信頼関係はばっちりですね！」
 "I-I see. What a coincidence.
  I-I also happen to feel the same way!
- I guess that means we trust each other completely, doesn't it!"
+ I guess that means we trust each other completely, doesn't it?!"
```

Comment:
Here too, unsure whether the tone of ?! is right, or if that's just how I interpret it.